src/extractor/guidance/turn_handler.cpp

```diff
@@ -227,26 +215,6 @@ Intersection TurnHandler::handleComplexTurn(const EdgeID via_edge, Intersection
                                      via_edge,
                                      isThroughStreet(obvious_index, intersection),
                                      intersection[obvious_index]);
-        if (has_same_name_turn &&
```

Comment:
Can you give me more context on why this could be removed? The only change I see that could be connected to this would be changes in `findObviousTurn()`, but I'm not yet clear why they are connected.